tests/model_serving/model_runtime/vllm/speculative_decoding/test_granite_7b_lab_draft.py

```diff
@@ -17,8 +17,7 @@
     "--model=/mnt/models/granite-7b-instruct",
     "--uvicorn-log-level=debug",
     "--dtype=float16",
-    "--speculative-model=/mnt/models/granite-7b-instruct-accelerator",
-    "--num-speculative-tokens=5",
+    '--speculative_config={"model": "/mnt/models/granite-7b-instruct-accelerator", "num_speculative_tokens": 5}',
```

Comment:
```suggestion
       "--speculative_config",
    "{"model": "/mnt/models/granite-7b-instruct-accelerator","num_speculative_tokens": 5}",
```

Reply:
@tarukumar 
 There is something wrong with the pre-commit hook configured on opendatahub-tests repo. I had made all changes as per your suggestion but pre-commit hook make a new commit on my PR and change the value.
https://github.com/opendatahub-io/opendatahub-tests/pull/288/commits/39314d0dadcdd091db27161610fc6646be0440f6